### ISSUE TITLE
Replace Player Inop After Loading From A Save In Lobby

### ISSUE THREAD

**Issue by BLOODWOLF333** [MegaMek/megamek#375]
HQ0.41.25 I setup a scenario and then saved just as a precaution because Im worried about if the resolution bug is fixed in 0.41.25 which Im about to test. I screwed up and forgot to give the opfor turrets so /reset. Then /load ERB and it set the opfor as a ghost. File>replace players was greyed out, Added a 3rd bot and could not transfer the ghost's units over to #3. /kick 2 and it then transferred her units but, still listed Simulation[Ghost}. Could not configure any of her units. /who listed only me the player, replacePlayer 2 -b:Princess -c:SIM failed since there was no player 2.  And loading a game does not restore your planetary conditions and generated map settings that i had configured before saving.

Player name: AFFS-FSAC-2REG-2BN-ECO (sorry Arlith :P)
[MMreplaceplayeronload.zip](https://github.com/MegaMek/megamek/files/543362/MMreplaceplayeronload.zip)

**Comment by Dylan-M** [comment]
As explained on the HQ ticket: copying some files in mmconf will cause this to happen because they remember this information on purpose.

**Comment by BLOODWOLF333** [comment]
What?

**Comment by Dylan-M** [comment]
Misread last night, thought this was a repeat of the mmconf folder loading issue. Re-opened.

**Comment by arlith** [comment]
The issue here is that you weren't really intended to be able to save/load games that are in the lounge.  The File>Game>Save menu items are purposefully disabled in the lounge.  It turns out that the `/save` command has no limitations on when it can be used, so it allows you to save a game in the lounge.

To be honestly, while it may seem a little odd, I feel that there is no reason we shouldn't allow saving games in the lobby (in fact, it almost seems like a beneficial client option would be to automatically save a game in the lobby before any game begins, just in case you screw something up and want to go back).

Basically, saving a game just serializes the current `Game` instance, and loading it restores it.  I'm able to load the saved game, and saving it worked, so I don't see a reason to not allow it.

Given that, the fix for this is easy: I can just remove the condition that disallows saving and replace player during the lounge phase (the /replacePlayer command did work on this saved game, you just have to do it before adding a bot or kicking another player).
